### PR TITLE
Adding required changes for EKS deployment.

### PR DESCRIPTION
**Pull request by bikramnehra**

### CODE REVIEW

File: backend/eks/clean.sh [hunk 1]
```diff
@@ -3,12 +3,17 @@
...
+. ./defaults.sh
 . ../../include/common.sh
 
-if [ -d "$BUILD_DIR" ]; then
+if [ -d "${BUILD_DIR}" ]; then
     . .envrc
-
-
+    if [ -f "${TFSTATE}" ]; then
+        mkdir -p cap-terraform/aks
```
Comment (viccuad) [inline backend/eks/clean.sh:12]: I suppose this should be eks, not aks

File: backend/eks/deploy.sh [hunk 2]
```diff
@@ -14,22 +14,28 @@ if ! aws sts get-caller-identity ; then
     aws configure
 fi
 
-git clone https://github.com/SUSE/cap-terraform.git -b cap-ci
+git clone https://github.com/SUSE/cap-terraform.git -b bisingh/eks
```
Comment (viccuad) [inline backend/eks/deploy.sh:17]: Could we merge bisingh/eks into cap-ci and consume that?

File: backend/eks/clean.sh (hunk 1 above)
Comment (bikramnehra) [reply backend/eks/clean.sh]: thanks done.

File: backend/eks/deploy.sh (hunk 2 above)
Comment (bikramnehra) [reply backend/eks/deploy.sh]: changes merged.

File: backend/eks/clean.sh [hunk 3]
```diff
@@ -3,12 +3,19 @@
 # Requires:
 # - aws credentials present
 
+. ./defaults.sh
 . ../../include/common.sh
 
-if [ -d "$BUILD_DIR" ]; then
-    . .envrc
-
+set -exuo pipefail
```
Comment (viccuad) [inline backend/eks/clean.sh:9]: This shouldn't be needed, it's set by include/common.sh

File: backend/eks/deploy.sh [hunk 4]
```diff
@@ -21,14 +21,21 @@ pushd cap-terraform/eks || exit
...
-hosted_zone_policy_arn = "${EKS_ZONE_POLICY}"
+cluster_name = "${EKS_CLUSTER_NAME}"
+region = "${EKS_LOCATION}"
+keypair_name = "${EKS_KEYPAIR}"
+eks_version = "${EKS_VERS}"
+cluster_labels = ${EKS_CLUSTER_LABEL}
+hosted_zone_name = "${EKS_HOSTED_ZONE_NAME}"
+external_dns_aws_access_key = "${AWS_ACCESS_KEY_ID}"
+external_dns_aws_secret_key = "${AWS_SECRET_ACCESS_KEY}"
+instance_type = "t2.large"
```
Comment (viccuad) [inline backend/eks/deploy.sh:32]: This was removed by https://github.com/SUSE/catapult/pull/234. Do we need it back?

**Pull request merged**